services/organisation/organisation.go

```diff
@@ -124,14 +133,31 @@ func UpdateOrganisation(orgId string, userId string, updateReq models.CreateOrgR
 		}
 	}
 
-	org.Name = updateReq.Name
-    org.Description = updateReq.Description
-    org.Email = updateReq.Email
-    org.State = updateReq.State
-    org.Industry = updateReq.Industry
-    org.Type = updateReq.Type
-    org.Address = updateReq.Address
-    org.Country = updateReq.Country
+	// Update the organization fields
+	if updateReq.Name != "" {
+		org.Name = updateReq.Name
+	}
+	if updateReq.Description != "" {
+		org.Description = updateReq.Description
+	}
+	if updateReq.Email != "" {
+		org.Email = updateReq.Email
+	}
+	if updateReq.State != "" {
+		org.State = updateReq.State
+	}
+	if updateReq.Industry != "" {
+		org.Industry = updateReq.Industry
+	}
+	if updateReq.Type != "" {
+		org.Type = updateReq.Type
+	}
+	if updateReq.Address != "" {
+		org.Address = updateReq.Address
+	}
+	if updateReq.Country != "" {
+		org.Country = updateReq.Country
+	}
```

Comment:
use the update function in the Postgres repository, or create a function that binds to the org struct
check other models for example